storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2378,25 +2372,28 @@ static int rocksdb_close_connection(handlerton *const hton, THD *const thd) {
  * be used as a rocksdb transaction name
  */
 static std::string rdb_xid_to_string(const XID &src) {
-  DBUG_ASSERT(src.gtrid_length >= 0 && src.gtrid_length <= MAXGTRIDSIZE);
-  DBUG_ASSERT(src.bqual_length >= 0 && src.bqual_length <= MAXBQUALSIZE);
+  DBUG_ASSERT(src.get_gtrid_length() >= 0 &&
```

Comment:
I see, I am not sure if sorting out MAX...SIZE visibility wouldn't be too much of a yak to shave, in which case it's fine to leave it alone. Re. passing the max values - would anyone ever pass different values than these constants?